system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -292,8 +291,6 @@ impl pallet_assets::Config<TrustBackedAssetsInstance> for Runtime {
 
 parameter_types! {
 	pub const AssetConversionPalletId: PalletId = PalletId(*b"py/ascon");
-	pub const AllowMultiAssetPools: bool = false;
-	// should be non-zero if AllowMultiAssetPools is true, otherwise can be zero
 	pub const LiquidityWithdrawalFee: Permill = Permill::from_percent(0);
```

Comment:
Given the comment you have deleted, is this still okay to have here `0`?

Reply:
Yeah this can be `0`